### PR TITLE
cleanup 

### PR DESCRIPTION
**Pull request by johba37**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +53,7 @@
     "webpack": "^3.3.0"
   },
   "dependencies": {
+    "aws-sdk": "^2.390.0",
```
Comment (troggy) [inline package.json]: do we need to bundle aws-sdk for lambda? I believe it should be globally available in AWS environment.

File: package.json [hunk 2]
```diff
@@ -56,6 +53,7 @@
...
     "babel-plugin-transform-builtin-extend": "^1.1.2",
```
Comment (troggy) [inline package.json]: no need in `babel-*` and `dashdash` deps — will make bundle smaller

File: package.json (hunk 1 above)
Comment (johba37) [reply package.json]: true

File: package.json (hunk 2 above)
Comment (johba37) [reply package.json]: i was having trouble running the function without `babel-runtime` actually.

Comment (johba37) [reply package.json]: oh, you are right, works without babel

File: package.json [hunk 3]
```diff
@@ -45,20 +42,12 @@
     "mocha-junit-reporter": "^1.17.0",
     "node-lambda": "^0.8.3",
```
Comment (troggy) [inline package.json]: not used

File: package.json [hunk 4]
```diff
@@ -45,20 +42,12 @@
...
     "prompts": "^0.1.8",
```
Comment (troggy) [inline package.json]: not used

**Review by troggy — APPROVED** [review]

**Pull request merged**